docs/docsite/rst/mazer/index.rst

```diff
@@ -9,21 +9,20 @@ Mazer is a new command-line tool for managing `Ansible <https://github.com/ansib
 on GitHub at `ansible/mazer <https://github.com/ansible/mazer>`_.
 
 .. note::
-    
+
     Mazer is experimental, and currently only available for tech-preview. Use with lots of caution! It is not intended for use in
     production environments, nor is it currently intended to replace the `ansible-galaxy` command-line tool.
 
     If you're installing Ansible content in a production environment, or need assistance with Ansible, please visit the `Ansible Project <https://github.com/ansible/ansible>`__,
     or the `Ansible docs site <https://docs.ansible.com>`_.
 
 .. note::
-    
-    Before installing roles with Mazer, review :ref:`using_mazer_content`. Mazer installs content different from
+
+    Before installing roles with Mazer, review :ref:`using_collections_in_playbooks`. Mazer installs content different from
```

Comment:
Mazer no longer install Roles. This should be re-worded, me thinks.

Reply:
updated